### PR TITLE
Refactoring - Hydra

### PR DESCRIPTION
**Pull request by meilame-tayebjee**
Refactored repo "from scratch", to match Hydra framework
Changed / simplified preprocessing : systematically add true codes, add textual add variabels at the end of libelle
train.py is runnable with torchFastText. 

To do next:
- post training handling (load from checkpoint etc)
- doc on how to add a new pytorch model
- mlflow integration
- adapt for fasttext meta
- adapt for camembert / transformers
- remove unnecessary files
- optional: think about abstract classes

### CODE REVIEW

**Comment by meilame-tayebjee** [comment]
- mlflow integration done in latest commits

Updated to-do: 

- change MLProject and adapt argo-workflows template
- evaluation on label studio data
- adapt for fasttext meta
- doc on how to add a new pytorch model
- adapt for camembert / transformers
- remove unnecessary files

**Comment by meilame-tayebjee** [comment]
argo workflows and MLProject updated
Warning: for now only works for torchFastText, but not fastText meta and CamemBERT.
To be continued on new branch

Updated to-do, by decreasing order of priority:

1. adapt for fasttext meta (train, mlflow, argo)
2. adapt for camembert / transformers
3. doc on how to add a new pytorch model
4. evaluation on label studio data
5. remove unnecessary files
